### PR TITLE
Issue 348 remote env drishya

### PR DESCRIPTION
**Pull request by drishyatm**
We have implemented the remote_credentials.py as env.remote 

Changed files : 

.circleci/config.yml
conf/browser_os_name_conf.py
conf/remote_credentials.py
page_objects/Base_Page.py
page_objects/driverfactory.py
page_objects/drivers/remote_options.py
utils/BrowserStack_Library.py
utils/interactive_mode.py

Added file 

env_remote   as replacement for remote_credentials_conf

Deleted file 
conf/remote_credentials.py

How to test : 

Switch to branch 
Rename the file env_remote as ".env.remote"
Provide the BS or SL details into it 
Execute /Run the pytest 
you could see the browserstack link with the test running on it.

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -119,7 +119,9 @@ a) Directory structure of our current Templates
 
    ./
 
-	|__conf: For all configurations and credential files
+	|__conf: For all configurations 
+
+	|__env For credential files
```
Comment (rohandudam) [inline Readme.md:124]: Are we creating new directory "env" and keeping .env file there?

File: Readme.md [hunk 2]
```diff
@@ -138,6 +140,16 @@ a) Directory structure of our current Templates
 COMMANDS FOR RUNNING TESTS
 ---------------------------
 
+Templates for.env files are accessible.
```
Comment (rohandudam) [inline Readme.md:143]: Good to move this session under Prerequisites or create new session after prerequisites. You may title it as "Configure environmental variables"

File: Readme.md (hunk 1 above)
Comment (drishyatm) [reply Readme.md:124]: No. Do you say , no need to mention about the ".env" in structure ? env is not as folder , I was mentioning it as file

Comment (rohandudam) [reply Readme.md]: Here your representation shows at env. Please remove it from structure

Comment (drishyatm) [reply Readme.md]: already did! Latest change will not show env in structure

File: .circleci/config.yml [hunk 3]
```diff
@@ -17,7 +17,7 @@ jobs:
 
         - run: git clone https://github.com/qxf2/bitcoin-info.git
 
-        - run: openssl aes-256-cbc -d -md sha256 -in ./conf/remote_credentials_enc.py -out ./conf/remote_credentials.py -pass env:KEY
```
Comment (rohandudam) [inline .circleci/config.yml:20]: Delete remote_credentials_enc.py file also.

Comment (drishyatm) [reply .circleci/config.yml:20]: Done! Fixed all feedbacks !

**Review by rohandudam — APPROVED** [review]

**Pull request merged**